test/driver_turso_sync/sqlc.yaml

```
@@ -0,0 +1,259 @@
+            - column: test_type_override.text_test
+              py_type:
+                import: collections
+                package: UserString
+                type: UserString
+            - column: test_override_conversion.price
+              py_type:
+                type: float
+            - db_type: JULIANDAY
+              py_type:
+                type: str
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)